readme.txt

```diff
@@ -0,0 +1,237 @@
+=== Newspack Newsletters ===
+Contributors: automattic, rabberson, adamboro, thomasguillot, dkoo, laurelfulford, claudiulodro, iuravic
+Requires at least: 5.3
+Tested up to: 5.5.1
+Requires PHP: 5.6
+Stable tag: trunk
+Tags: newsletters, Newspack, WordPress.com, Mailchimp, Constant Contact
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+Create email newsletters with the Gutenberg editor and distribute them with your Mailchimp or Constant Contact mailing lists.
+
+== Description ==
+Create email newsletters with the Gutenberg editor and send them via Mailchimp or Constant Contact, all without leaving WP Admin! Newspack Newsletters lets you build eye-catching newsletters using the WordPress editing tools you’re already familiar with, and lets you save drafts, create reusable layouts, send to your existing mailing list, and also publish them to your website.
+
+= Use and create newsletter layouts =
+
+Newspack Newsletters comes with four default layouts to help you start building newsletters quickly. You can use an existing layout, or start fresh with a blank page. You can also create and save your own layouts, or alter the ones that come bundled with the plugin and save the settings, and use them for future campaigns.
+
+= Use Gutenberg blocks and settings =
+
+Newspack Newsletters supports most core Gutenberg blocks and configurations, letting you use blocks like Group, Button, Columns, and more, to create dynamic layouts. The plugin will warn you when a block will not display as expected in an email. Newspack Newsletters uses [MJML](https://mjml.io/) to transform the HTML generated by Gutenberg into markup that will be displayed nicely by email clients.
+
+= Customize your newsletter’s appearance =
+
+You can customize your newsletters further by changing the header and body fonts, or picking a background color.
+
+= Newspack Blocks =
+
+Newspack Newsletters comes bundled with the Newspack Post Inserter block. This block lets you insert excerpts from live posts on your site, and, once inserted, will turn those posts into static content using the Header and Paragraph blocks, linking back to your site.
+
+Need to target a post title to your newsletter audience? Not a problem! You can edit these blocks, and add others, to make sure they’ speak to your readers.
+
+= Add ads! =
+
+Monetize your newsletters by including advertising. The Newspack Newsletters plugin lets you create ads, which are stored as a custom post type. Ads are automatically appended to the end of your newsletters, and can also be toggled off on a newsletter-by-newsletter basis.
```

Comment:
Ads can be set to a percentage position ;)

```suggestion
Monetize your newsletters by including advertising. The Newspack Newsletters plugin lets you create ads, which are stored as a custom post type. Ads are automatically inserted into your newsletters, and can also be toggled off on a newsletter-by-newsletter basis.
```

Reply:
😮 I did not know this! That's awesome! 

Updated as suggested in cb7af53